docs/releases/1.10.txt

```diff
@@ -301,7 +301,7 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* Added methods :meth:`~django.test.testcases.SimpleTestCase.assertFormValid` and :meth:`~django.test.testcases.SimpleTestCase.assertFormNotValid`.
```

Comment:
omit ".testcases" -- the documentation references use the convenience import path to `SimpleTestCase`.